administration/src/bp-modules/stores/StoresController.tsx

```diff
@@ -22,10 +20,12 @@ const StoresController = (): ReactElement => {
           ? t('errors:notAuthorizedToManageStores')
           : t('errors:manageStoresNotActivated'),
       }}>
-      <Stack sx={{ flexGrow: 1, alignItems: 'center', justifyContent: 'safe center' }}>
-        <Button startIcon={<CsvIcon />} onClick={() => navigate('./import')}>
-          {t('storesCsvImport')}
-        </Button>
+      <Stack sx={{ flexGrow: 1, alignItems: 'center', justifyContent: 'safe center', overflowY: 'auto' }}>
+        <Stack sx={{ width: '400px', padding: 2, gap: 2 }}>
+          <Button size='large' startIcon={<CsvIcon />} href='./import' sx={{ flexDirection: 'column', gap: 1, py: 2 }}>
+            {t('storesCsvImport')}
+          </Button>
+        </Stack>
```

Comment:
❓ Do we really need this inner `<Stack>` here? Wouldn't setting `width` on the button suffice?